scripts/kubectl-kubestellar-ensure-location

```diff
@@ -108,13 +108,25 @@ fi
 
 set -e
 
+echo "--- current directory is $PWD"
+cwsi=$(kubectl ws "${kubectl_flags[@]}" .)
```

Comment:
What does "cwsi" stand for?

Reply:
current workspace "cws" I don't remember what "i" was for, if anything.